tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -383,7 +383,7 @@ public function testJsonbColumn(string $type): void
         /** @var Schema\Column[] $columns */
         $columns = $this->_sm->listTableColumns('test_jsonb');
 
-        self::assertSame(TYPE::JSON, $columns['foo']->getType()->getName());
+        self::assertSame($type, $columns['foo']->getType()->getName());
```

Comment:
Is this test change required? Smells like even more BC breaks

Reply:
It's required. It's part of the BC break fix